src/tutorial/errors.md

```diff
@@ -233,3 +233,12 @@ Error: could not read file `test.txt`
 Caused by:
     No such file or directory (os error 2)
 ```
+
+## Wrapping up
+
+The complete code for our `grrs` tool with improved error reporting
```

Comment:
Let's be consistent with the sentences in this section. Please remove this line in favor of "Your code should now look like:"

Reply:
I will do that. But note that I actually did not use that phrase on purpose, because the `grrs` code is not developed incrementally into the given form. Rather is the example used in the error chapter essentially independent from the code developed earlier (admittedly with some similarities).

That's IMO an issue that should also be addressed at some time.